login-workflow/src/new-architecture/components/ChangePasswordDialog/types.ts

```diff
@@ -1,14 +1,19 @@
-import { DialogProps } from '@mui/material';
+import { SxProps } from '@mui/material';
+import { BasicDialogProps } from '../Dialog';
 import { SetPasswordProps } from '../SetPassword';
 
-// we should define a reusable type for the password input props (this should be defined with the reusable password component and reused by CreatePassword, ResetPassword, and ChangePassword).
-export type ChangePasswordDialogProps = DialogProps & { PasswordProps: SetPasswordProps } & {
+export type ChangePasswordDialogProps = { passwordProps?: SetPasswordProps } & {
```

Comment:
We need `DialogProps` for `ChangePasswordDialogProps`, as the root comp is Dialog, it should accept all the props of dialog.